config/locales/candidate_interface/degree.yml

```
@@ -151,3 +166,46 @@ en:
               invalid_year: Enter a real graduation year
               in_the_future: Enter a year that is the current year or a year in the future
               in_time_for_training: The date you graduate must be before the start of your teacher training
+        candidate_interface/degree_wizard:
+          attributes:
+            uk_or_non_uk:
+              blank: Select whether this is a UK or a non-UK degree
```

I think this error message should mirror the labels of the radios more closely? cc: @EmmaFrith 
```suggestion
              blank: Select if the degree was from the United Kingdom or another country
```